next.config.js

```diff
@@ -0,0 +1,3 @@
+const withCSS = require('@zeit/next-css');
```

Comment:
This doesn't seem to be needed anymore with the way we use bulm in static assets.